RazorGenerator.Mvc/PrecompiledMvcEngine.cs

```diff
@@ -33,53 +33,46 @@ public PrecompiledMvcEngine(Assembly assembly, string baseVirtualPath)
         {
         }
 
-        public PrecompiledMvcEngine(Assembly assembly, string baseVirtualPath, IViewPageActivator viewPageActivator)
+        public static string[] DefaultAreaLocationFormats()
         {
-            AssemblyLastWriteTime = new Lazy<DateTime>(() => assembly.GetLastWriteTimeUtc(fallback: DateTime.MaxValue));
-            BaseVirtualPath = NormalizeBaseVirtualPath(baseVirtualPath);
-
-            base.AreaViewLocationFormats = new[] {
-                "~/Areas/{2}/Views/{1}/{0}.cshtml", 
-                "~/Areas/{2}/Views/{1}/{0}.vbhtml", 
-                "~/Areas/{2}/Views/Shared/{0}.cshtml", 
-                "~/Areas/{2}/Views/Shared/{0}.vbhtml", 
+            return new[] {
+                "~/Areas/{2}/Views/{1}/{0}.cshtml",
+                "~/Areas/{2}/Views/{1}/{0}.vbhtml",
+                "~/Areas/{2}/Views/Shared/{0}.cshtml",
+                "~/Areas/{2}/Views/Shared/{0}.vbhtml",
             };
+        }
 
-            base.AreaMasterLocationFormats = new[] {
-                "~/Areas/{2}/Views/{1}/{0}.cshtml", 
-                "~/Areas/{2}/Views/{1}/{0}.vbhtml", 
-                "~/Areas/{2}/Views/Shared/{0}.cshtml", 
-                "~/Areas/{2}/Views/Shared/{0}.vbhtml", 
+        public static string[] DefaultLocationFormats()
```

Comment:
Could you make these regular properties or readonly fields? It's a bit unusual to see methods that don't follow the Get \ Set \ Do* format.